tests/pytest/pems/test_routes.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from pems.routes import Routes, routes
+
+
+@pytest.mark.parametrize(("route", "expected_name"), [("app1:name", "name"), ("app2:index", "index")])
+def test_name_route(route: str, expected_name: str):
+    assert routes.name(route) == expected_name
```

Comment:
Since `name()` is a `@staticmethod` of the `Routes` class, I feel like it would make more sense to run the test from the `Routes` class directory vs. this instance.